### PR TITLE
CI: Accessibility scan

### PR DESCRIPTION
**Pull request by SarahJaine**
**Description of change**
Add accessibility scan of all urls to continuous integration pipeline

Accessibility issues were found by the scan and will take some investigation to fix. I created a new issue, https://github.com/HHS/Head-Start-TTADP/issues/159, for this work and to add the `--exit` flag into the axe command. The exit flag will cause the job to fail when an accessibility issue is found.

**How to test**
- See axe accessibility scan run in [recent circle ci job on fork's main](https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/667/workflows/6a11ff16-119e-4f9b-960e-bc2cd5e25a3c/jobs/2899)

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/41

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated - **_I simply mentioned this in the README, do you think we need an ADR?_**
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ module.exports = {
     database: process.env.POSTGRES_DB,
     host: process.env.POSTGRES_HOST,
     dialect: 'postgres',
+    logging: false,
```
Comment (SarahJaine) [inline config/config.js:17]: Whoops, this slipped in, I can take this out next time I add a feature. I don't think anyone will really miss the default of extra verbose sequelize log statements in the meantime.

File: .circleci/config.yml [hunk 2]
```diff
@@ -245,6 +245,29 @@ jobs:
...
+          command: yarn build
+      - run:
+          name: Start server
+          command: BYPASS_AUTH=true yarn start:ci
+          background: true
+      - run:
+          name: Run axe
+          command: |
+            mkdir reports
+            sleep 5
```
Comment (rahearn) [inline .circleci/config.yml:267]: Why is the sleep necessary? To let the server have time to start up?

File: src/middleware/authMiddleware.js [hunk 3]
```diff
@@ -45,6 +39,10 @@ export function login(req, res) {
  */
 
 export default async function authMiddleware(req, res, next) {
+  // bypass authorization, used for cucumber UAT and axe accessibility testing
+  if (process.env.NODE_ENV !== 'production' && process.env.BYPASS_AUTH === 'true') {
```
Comment (rahearn) [inline src/middleware/authMiddleware.js:43]: Glad to see the additional `NODE_ENV` check 👍🏻

**Review by rahearn — APPROVED** [review]
Great work. Can you just verify my question about the call to `sleep`?

File: .circleci/config.yml (hunk 2 above)
Comment (SarahJaine) [reply .circleci/config.yml:267]: This one hurt a little. Unfortunately, yes, the server needs time to start up. The axe command fails without it.

File: src/middleware/authMiddleware.js (hunk 3 above)
Comment (SarahJaine) [reply src/middleware/authMiddleware.js:43]: I'm nothing if not paranoid :)

**Pull request merged**